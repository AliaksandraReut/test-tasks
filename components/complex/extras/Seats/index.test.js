import React from 'react';
import seatsFactory from './index.component';
import renderer from 'react-test-renderer';

const SeatMock = () => <span />;
const TransatSeatMapMock = () => <span />;

const Seats = seatsFactory({
	Seat: SeatMock,
	TransatSeatMap: TransatSeatMapMock,
});

describe('Seats Component tests', () => {
	it('Seats should match snapshot', () => {
		const component = renderer.create(
			<Seats />
		);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
