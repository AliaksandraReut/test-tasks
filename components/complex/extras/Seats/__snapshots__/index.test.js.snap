// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seats Component tests Seats should match snapshot 1`] = `
<div
  className="seats-container"
>
  <span />
</div>
`;
